modules/exploits/multi/http/phpstudy_backdoor_rce.rb

```diff
@@ -0,0 +1,67 @@
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info={})
+    super(update_info(info,
+      'Name'           => "Phpstudy Backdoor Remote Code execution",
+      'Description'    => %q{
+        This module can detect and exploit the backdoor of PHPStudy.
+      },
+      'License'        => MSF_LICENSE,
+      'Author'         => [ 'Airevan' ],
```

Comment:
Who discovered the vulnerability?

Do you have any reference URLs? CVE?

Reply:
The backdoor is because the development team was hacked

Reply:
no cve id

Reply:
I remember. Usually the person who discovered and/or analysed the backdoor are credited. Also, whoever wrote PoC code or exploits for the backdoor.

Also, please add reference URLs.